src/optimize.jl

```diff
@@ -0,0 +1,118 @@
+
+function pm_next!(pm, stats::NamedTuple)
+    ProgressMeter.next!(pm; showvalues=[tuple(s...) for s in pairs(stats)])
+end
+
+"""
+    optimize(
+        prob,
+        objective    ::AbstractVariationalObjective,
+        restructure,
+        λ₀           ::AbstractVector{<:Real},
+        n_max_iter   ::Int,
+        objargs...;
+        kwargs...
+    )              
+
+Optimize the variational objective `objective` targeting `prob` by estimating (stochastic) gradients, where the variational approximation can be constructed by passing the variational parameters `λ₀` to the function `restructure`.
+
+    optimize(
+        prob,
+        objective ::AbstractVariationalObjective,
+        q,
+        n_max_iter::Int,
+        objargs...;
+        kwargs...
+    )              
```

Comment:
Same here regarding putting the method definitions just next to each other

Reply:
done.